### ISSUE TITLE
Contents of volume are deleted in version 2.1.36 and later

### ISSUE THREAD

**Issue by muskedunder** [adobe/S3Mock#286]
I create an s3 container according to the following compose file, then I post some data to the container.
```
version: '3.7'
services:
  s3:
    image: adobe/s3mock:2.1.36
    ports:
      - "9090:9090"
    environment:
      - initialBuckets=myFirstBucket
      - root=/data
    volumes:
      - s3_data:/data
volumes:
  s3_data:
```
I can inspect the volume `s3_data` and see that the data is there as expected.

However, if I stop the container the data in the volume is suddenly lost, which was not the case with version 2.1.34.

**Comment by afranken** [comment]
Yes, that is correct.

Before, S3Mock would never delete any files, cluttering every developer's machine or build system.
Now it deletes any files that were created during it's lifetime.

**Comment by muskedunder** [comment]
Hmm, ok.

But if I explicitly create a volume that maps to the data directory of the container I would certainly expect it to remain even after the container is stopped, and that is also how it works for all other database containers that I have used.

**Comment by afranken** [comment]
@muskedunder Just released 2.2.2 containing this change.
